src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/my_db";
```

Comment:
The database URL seems to be incomplete. Make sure to provide the full JDBC URL for the database connection.